SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+When you find a vulnerability, please report it by emailing to eduvpn@surfnet.nl
+
+Please use proper disclosure proces. Most likely we will ask you to create a pull request, or we create one based on your input.
```

Comment:
Typo: "proces" -> "process".

It's unclear to me what the "proper disclosure process" is here. Maybe add a link or something?

Reply:
Typo fixed and based info on that what is mentioned on eduvpn.org.